controllers/dpa_controller.go

```diff
@@ -64,10 +64,10 @@ var debugMode = os.Getenv("DEBUG") == "true"
 //+kubebuilder:rbac:groups=oadp.openshift.io,resources=dataprotectionapplications/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=oadp.openshift.io,resources=dataprotectionapplications/finalizers,verbs=update
```

Comment:
do you wanna add delete permission as a kubebuilder tag here too?

Reply:
for reference: [RBAC](https://book.kubebuilder.io/reference/markers/rbac) markers

Reply:
If you add rbac markers I think when you run `make generate` it will add it to the role.yaml for you or something along those lines.

Reply:
but doing it manually also works.

Reply:
Pretty sure rbac markers are additive and only adds more to the existing role. It does not delete perms without markers that were added before.